### PR TITLE
Add new ivory anatomical entity pattern templates

### LINKED ISSUE #823: Add new abnormally ivory anatomical entity pattern templates
[PATO:0002149 ivory](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0002149) is a
a quality inhering in a bearer by virtue of the bearer exhibiting increased density resulting in exhibiting **increased degree of radiopacity (in X-rays).**

1. [x] add new pattern `abnormallyIvoryAnatomicalEntity.yaml` for review
* Examples:
    1. HP:0010583 [Ivory epiphyses](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0010583)
    2. HP:0009191 [Ivory epiphyses of the metacarpals](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0009191)
    3. HP:0010234 [Ivory epiphyses of the phalanges of the hand](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0010234)
2. [x] add new pattern `abnormallyIvoryAnatomicalEntityInLocation.yaml` for review
* Examples:
    1. HP:0010119 [Ivory epiphyses of the hallux](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0010119)
    2. HP:0010168 [Ivory epiphyses of the toes](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0010168)
    3. HP:0009388 [Ivory epiphyses of the 5th finger](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0009388)
3. [x] get feedback
4. [ ] fix or improve pattern based on feedback

### PR DESCRIPTION
**Pull request by rays22**
If applied, this commit will fix #823.

### CODE REVIEW

**Comment by sbello** [comment]
fine with me: orcid.org/0000-0003-4606-0597

**Comment by anna-anagnostop** [comment]
Ok with me Anna Anagnostopoulos https://orcid.org/0000-0002-6490-7723

**Comment by matentzn** [comment]
ok by me https://orcid.org/0000-0002-7356-1779

**Comment by LCCarmody** [comment]
Okay. https://orcid.org/
0000-0001-7941-2961

File: src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+pattern_name: abnormallyIvoryAnatomicalEntity
+
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormallyIvoryAnatomicalEntity.yaml
+
+description: 'Please, note that the term "ivory" in the context of this phenotype pattern refers to the increased density of an anatomical entity (e.g. epiphysis) observable by radiography (X-rays).'
```
Comment (rays22) [inline src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml:6]: Strip off 'Please, note...'

File: src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntityInLocation.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+pattern_name: abnormallyIvoryAnatomicalEntityInLocation
+
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormallyIvoryAnatomicalEntityInLocation.yaml
+
+description: 'Please, note that the term "ivory" in the context of this phenotype pattern refers to the increased density of an anatomical entity (e.g. epiphysis) observable by radiography (X-rays). Use this pattern for anatomical locations for which the anatomical class is composed of two anatomical terms (e.g. "Ivory epiphyses of the toes").'
```
Comment (rays22) [inline src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntityInLocation.yaml:6]: Strip off 'Please, note...'

File: src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+def:
+  text: "An abnormal increase in the radiographic density of %s."
+  vars:
+    - anatomical_entity
+
+equivalentTo:
+  text: "'has_part' some (
+            'ivory' and
+            'characteristic_of' some %s and
+            'has_modifier' some 'abnormal')"
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml:52]: "'has_part' some (
            'ivory' and
            ('characteristic_of' some %s) and
            ('has_modifier' some 'abnormal'))"

Better use brackets () to ensure  correct unin

File: src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml (hunk 1 above)
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml:6]: remove please note and consider standard pattern description text.

File: src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
+pattern_name: abnormallyIvoryAnatomicalEntity
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml:2]: drop abnormal from pattern name

File: src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormallyIvoryAnatomicalEntity.yaml
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml:4]: drop abnormally from the pattern name, just ivoryAnatomicalEntity

File: src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    - anatomical_entity
+
+annotations:
+  - annotationProperty: exact_synonym
+    text: "abnormal radiographic density of %s"
+    vars:
+      - anatomical_entity
+
+def:
+  text: "An abnormal increase in the radiographic density of %s."
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml:44]: Make more close to pato def.

File: src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    - anatomical_entity
+
+equivalentTo:
+  text: "'has_part' some (
+            'ivory' and
+            'characteristic_of' some %s and
+            'has_modifier' some 'abnormal')"
+  vars:
+    - anatomical_entity
+...
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml:55]: what are the three dots there?

File: src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntityInLocation.yaml [hunk 8]
```diff
@@ -0,0 +1,66 @@
+---
+pattern_name: abnormallyIvoryAnatomicalEntityInLocation
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntityInLocation.yaml:2]: see above

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml (hunk 7 above)
Comment (rays22) [reply src/patterns/dosdp-dev/abnormallyIvoryAnatomicalEntity.yaml:55]: Three dots ( “...”) indicate the end of a [yaml 1.2.2 document without starting a new one.](https://yaml.org/spec/1.2.2/#22-structures)
For consistency with the other yaml files here, I will remove them.

**Pull request merged**